Formula/docker-machine.rb

```diff
@@ -36,6 +36,43 @@ def install
     end
   end
 
+  def caveats
+    s = <<-EOS.undent
+
+    Create a docker machine ie: docker-machine create -d {driver} default
+    EOS
+    s
+  end
+
+  plist_options :manual => "docker-machine start"
+
+  def plist; <<-EOS.undent
+     <?xml version="1.0" encoding="UTF-8"?>
+     <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+     <plist version="1.0">
+       <dict>
+         <key>EnvironmentVariables</key>
+         <dict>
+             <key>PATH</key>
+             <string>/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin</string>
```

Comment:
You presumably want `#{HOMEBREW_PREFIX}/bin` rather than `/usr/local/bin`. Homebrew isn't always in `/usr/local`.